hmda/src/main/scala/hmda/validation/rules/lar/validity/common/V631_1.scala

```diff
@@ -25,7 +25,7 @@ object V631_1 extends EditCheck[LoanApplicationRegister] {
   )
 
   override def apply(lar: LoanApplicationRegister): ValidationResult =
-    if(lar.coApplicant.ethnicity.otherHispanicOrLatino == "") {
+    if(lar.coApplicant.ethnicity.otherHispanicOrLatino.isEmpty) {
```

Comment:
This is replicating the `when` block in the previous logic using a the ethnicity string. This if implements these rules [here](https://ffiec.cfpb.gov/documentation/fig/overview#table2-25).